L1Trigger/Phase2L1ParticleFlow/python/l1pfJetMet_cff.py

```diff
@@ -1,29 +1,29 @@
 import FWCore.ParameterSet.Config as cms
 
-from L1Trigger.Phase2L1ParticleFlow.L1SeedConePFJetProducer_cfi import L1SeedConePFJetProducer, L1SeedConePFJetEmulatorProducer
-from L1Trigger.Phase2L1ParticleFlow.DeregionizerProducer_cfi import DeregionizerProducer as l1ctLayer2Deregionizer
-scPFL1PF            = L1SeedConePFJetProducer.clone(L1PFObjects = 'l1ctLayer1:PF')
-scPFL1Puppi         = L1SeedConePFJetProducer.clone()
-scPFL1PuppiEmulator = L1SeedConePFJetEmulatorProducer.clone(L1PFObject = cms.InputTag('l1ctLayer2Deregionizer:Puppi'))
+from L1Trigger.Phase2L1ParticleFlow.L1SeedConePFJetProducer_cfi import l1tSeedConePFJetProducer, l1tSeedConePFJetEmulatorProducer
+from L1Trigger.Phase2L1ParticleFlow.DeregionizerProducer_cfi import l1tDeregionizerProducer as l1tLayer2Deregionizer
+scPFL1PF            = l1tSeedConePFJetProducer.clone(L1PFObjects = 'l1ctLayer1:PF')
+scPFL1Puppi         = l1tSeedConePFJetProducer.clone()
+scPFL1PuppiEmulator = l1tSeedConePFJetEmulatorProducer.clone(L1PFObject = cms.InputTag('l1ctLayer2Deregionizer:Puppi'))
 
-_correctedJets = cms.EDProducer("L1TCorrectedPFJetProducer", 
+correctedJets = cms.EDProducer("L1TCorrectedPFJetProducer", 
```

Comment:
The leading underscore should remain in place if `correctedJets` is not really used in the final configuration (it avoids that `correctedJets` is imported from this file). Actually, here `correctedJets` is only used a few lines below to make `scPFL1PuppiCorrectedEmulator`, so why not just build and customise the latter (without any `correctedJets`)?

Reply:
I switched back to _correctedJets